lock-api-objects/src/main/java/com/palantir/lock/watch/IdentifiedVersion.java

```diff
@@ -25,5 +26,5 @@
     @Value.Parameter
     UUID id();
     @Value.Parameter
-    long version();
+    OptionalLong version();
```

Comment:
?

Reply:
We need to support unknown version

Reply:
probably a silly question simply because I'm not aware of lock watch intricacies, but why do we need to? this is all new code right?